docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -48,3 +45,12 @@ Runs with the browser in headful mode.
 *`--inline`*::
 Instead of reading from a file, `cat` inline scripted journeys and pipe them through `stdin`.
 For example, `cat path/to/file.js | npx @elastic/synthetics --inline`
+
+*`--screenshots`*::
+Enables capturing screenshots for every step in the journey.
+
+*`--network`*::
```

Comment:
Can we add the junit output options too?

Reply:
We already have the reporter option documented above.